src/ui/MissingTerm/MissingTermManager.ts

```diff
@@ -1,38 +1,55 @@
 import 'styling/_MissingTermsBreadcrumb';
 import { $$ } from '../../utils/Dom';
-import { QueryEvents, QueryStateModel, BreadcrumbEvents, l } from '../../Core';
+import { QueryEvents, QueryStateModel, BreadcrumbEvents, l, get } from '../../Core';
 import { IDoneBuildingQueryEventArgs } from '../../events/QueryEvents';
 import { IClearBreadcrumbEventArgs, IPopulateBreadcrumbEventArgs } from '../../events/BreadcrumbEvents';
 import { SVGIcons } from '../../utils/SVGIcons';
 import { QueryController } from '../../controllers/QueryController';
+import { MODEL_EVENTS, IAttributeChangedEventArg } from '../../models/Model';
+import { QUERY_STATE_ATTRIBUTES } from '../../models/QueryStateModel';
+import XRegExp = require('xregexp');
+import { Breadcrumb } from '../Breadcrumb/Breadcrumb';
 
 export class MissingTermManager {
   static ID = 'MissingTermManager';
+  // Used to split terms and phrases. Match character that can separate words or caracter for Chinese, Japanese and Korean.
+  // Han: Unicode script for Chinesse character
+  // We only need to import 1 Asian, charcaters script because what is important here is the space between the caracter and any script will contain it
```

Comment:
Some typos: characters*   character*
Not sure the sentence is clear, how about:
`We only need to import one Asian characters script because what is important here is the space between characters and any of those scripts will contain it.`

Reply:
👍  will do